.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,25 +1,34 @@
 <!-- Thank you for your pull request, please provide a brief summary of what this introduces (mandatory). Please point out any code that may be non-obvious to reviewers by using comments. -->
 
-<!-- Make sure that you've linted your files, written and run unit tests, and filled out or updated documentation (README) -->
-
 ## Summary
 
+Fixes: #1234 <!-- For bug fixes, use "Fixes". For new features use "Resolves". This helps link a PR to an issue and will show up in release notes. -->
+
 <!-- Explain the **motivation** for making this change. What existing problem does the pull request solve? -->
 
+<!-- This is the category in the release notes. Common categories are Components, Infrastructure, and Documentation -->
+### Category
+Components
```

Comment:
My thought is that the CI picks up on this category to know which category to group the PR to in the changelog.

Reply:
While good indicators for the CI script, I feel like these headers will look kind of random in the PR description. What do you think about using shields.io for these? We could have the `img` tag commented out and you just uncomment and edit as necessary

Reply:
Hmm. Like:
```
![category](https://img.shields.io/badge/category-Utils_&_Hooks-green)
```

![category](https://img.shields.io/badge/category-Utils_&_Hooks-green)

That could work. I like the look of that. That seems a little harder to edit than a section. It is also different than `Notes` and `BREAKING CHANGES`, but those are not going to be common.

Reply:
This is what it could look like:

Reply:
<!-- Thank you for your pull request, please provide a brief summary of what this introduces (mandatory). Please point out any code that may be non-obvious to reviewers by using comments. -->

![category](https://img.shields.io/badge/category-Components-green)

## Summary

Fixes: #1234 <!-- For bug fixes, use "Fixes". For new features use "Resolves". This helps link a PR to an issue and will show up in release notes. -->

Fixes `OverflowTooltip` with icons in IE11. IE11 doesn't have `children` on the `Element` interface which includes SVG elements, but provides it on the `HTMLElement` interface which does not include SVG elements. This fix checks for the presence of `children` before looping when looking for the overflow element.

<!-- Explain the **motivation** for making this change. What existing problem does the pull request solve? Anything in the Summary section will be attached to the squashed commit when this PR is merged. -->

### BREAKING CHANGES
Optional breaking changes message. If your PR includes breaking changes. It is extremely rare to put breaking changes outside a `prerelease/v*` branch. Anything in this section will show up in release notes. Remove this section if no breaking changes are present.

### Notes
Optional notes message. Anything in this section will show up in release notes.

---

## Checklist

<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->

- [x] tests are changed or added
- [x] code has been documented
- [x] design approved final implementation
- [x] a11y approved final implementation
- [x] code adheres to the [API & Pattern guidelines](https://workday.github.io/canvas-kit/?path=/story/welcome-dev-docs-api-pattern-guidelines--page)

## Additional References

<!-- Upload screenshots of the final component or any other artifacts that would help a reviewer understand the choices you made in the PR. -->

Reply:
@willklein Your thoughts on the following:

---

### Release Category
Components

---

vs

---

![category](https://img.shields.io/badge/release_category-Components-blue)

---

The badge markdown looks like this:
```
![category](https://img.shields.io/badge/release_category-Components-blue)
```

I think it definitely looks better when inlined with Summary and is easier to scan for.

Reply:
I prefer the badges by far.